scenario2/benchmark-job.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: batch/v1
```

Comment:
I feel like you shouldn't need a separate yaml for the scenario2 job, since it's the same as scenario1. Do you expect future scenarios to have different job yamls from each other?

Reply:
I vote for pushing reused functions that don't change often for different scenarios to a common folder as much as you can, and have each scenario script use those functions.